ansys/dpf/core/operators/averaging/elemental_difference.py

```diff
@@ -76,7 +91,7 @@ def default_config():
 
     @property
     def inputs(self):
```

Comment:
Alex says that a property is described as a noun, so I've made these changes for your review and approval.